src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/cas3/Cas3ReportsTest.kt

```diff
@@ -2342,7 +2342,7 @@ class Cas3ReportsTest : IntegrationTestBase() {
   }
 
   @Nested
-  inner class GetBedUtilizationReport {
+  inner class GetBedUtilReport {
```

Comment:
This class name results in class filenames which result in errors during build since the filenames are too long.  Reducing the class name length solves this issue.  Added more information to the PR description too.

Reply:
This can be solve by reduce the tests name not the class name

Reply:
See comment above.